messaging/src/main/java/org/axonframework/eventhandling/scheduling/jobrunr/JobRunrEventScheduler.java

```diff
@@ -290,6 +288,19 @@ public Builder jobScheduler(JobScheduler jobScheduler) {
             return this;
         }
 
+        /**
+         * Sets the {@link String} used for creating the scheduled jobs in JobRunr. Defaults to
+         * {@code AxonScheduledEvent}.
+         *
+         * @param jobName a {@link JobScheduler} used for naming the job
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public Builder jobName(String jobName) {
```

Comment:
I guess the job name is a hard requirement when using the `JobBuilder`? Furthermore, I assume these do not have to be unique, right?

Reply:
I don]t think it needs to be provided, but by default, it will fall back to the method to be called and some of the arguments, which can be pretty long. It doesn't need to be unique.